kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/SchemaUtilsSuite.scala

```diff
@@ -310,6 +311,180 @@ class SchemaUtilsSuite extends AnyFunSuite {
     }
   }
 
+  ///////////////////////////////////////////////////////////////////////////
+  // computeSchemaChangesById
+  ///////////////////////////////////////////////////////////////////////////
+  test("Compute schema changes with added columns") {
+    val fieldMappingBefore = Map(
+      1L -> new StructField("id", IntegerType.INTEGER, true)).map { case (k, v) =>
+      java.lang.Long.valueOf(k) -> v
+    }.asJava
+
+    val fieldMappingAfter = Map(
+      1L -> new StructField("id", IntegerType.INTEGER, true),
+      2L -> new StructField("data", StringType.STRING, true)).map { case (k, v) =>
+      java.lang.Long.valueOf(k) -> v
+    }.asJava
+
+    val schemaChanges = SchemaUtils.computeSchemaChangesById(fieldMappingBefore, fieldMappingAfter)
```

Comment:
just one suggestion to make the simplify the tests.

```
val fieldMappingBefore = Map(1L -> new StructField("id", IntegerType.INTEGER, true))
val fieldMappingAfter = Map(
      1L -> new StructField("id", IntegerType.INTEGER, true),
      2L -> new StructField("data", StringType.STRING, true))

val schemaChanges = computeSchemaChangesByIdHelper(fieldMappingBefore, fieldMappingAfter)
```

```
def computeSchemaChangesByIdHelper(before: Map[Long, StructField], after: Map[Long, StructField]): SchemaChanges = {
   // do all the scala to java stuff and call the utility
}
```

Reply:
Good suggestion, that reduced a bit in the tests.